### ISSUE TITLE
Auto-populate CC emails for ROW Deficiency Emails (SR)

### ISSUE THREAD

**Issue by atdservicebot** [cityofaustin/atd-data-tech#12104]
** Acceptance Criteria ** 

- The Acceptance Criteria provided below applies to RW, DS, EX and ECV folders with ATD subtype in AMANDA. 
- When user uses the Email/Print button on Folder Tab and selects ROW Deficiency - Email (50563) or ROW Open Deficiency-Email (50568) report, system should add the following email IDs to the CC of the email automatically. (Click Edit to view email details)
-- Nick M. 
-- Gary Z. 
-- Blake B. 
-- Darren U. 

> What application are you using?

AMANDA

> Describe the problem.

Our Inspections & Enforcement workgroup wants the email addresses for the following staff members to auto-populate in the CC field when they Send the "ROW Deficiency- Email" (50563) report and the "ROW Open Deficiency-Email" (50568) report:

- Nick M. 
- Gary Z. 
- Blake B. 
- Darren U. 

This applies to the ECV folder with the Austin Transportation Sub Type, as well as the DS, EX, RW and PSM folders.

> Describe the outcome you'd like to see when this feature is implemented.

Our Inspections & Enforcement workgroup will be able to more effectively and efficiently track deficiencies and follow up on them to ensure compliance.

> Describe any workarounds you currently have in place or alternative solutions you've considered.

Currently staff manually enters the email addresses for these staff members.

> Requested By
Kyle K.

Request ID: DTS23-107375

**Comment by vschandramouli** [comment]
Updated AC, moving to backlog.

**Comment by Sreelatha-BlazeKannan** [comment]
@Nadin-Nader , @TracyLinder - Analysis is in progress

**Comment by Nadin-Nader** [comment]
@Sreelatha-BlazeKannan this has been sent to the DevOps CM Team to be deployed to DEV and TEST under **SCTASK0318195**

**Comment by Nadin-Nader** [comment]
Still waiting on CTM to deploy

**Comment by Nadin-Nader** [comment]
@Sreelatha-BlazeKannan I received an email from Anu stating this is completed, Please UAT and let me know if it is all good

**Comment by TracyLinder** [comment]
### UAT IN TEST 

✅ RW
❌ EX - 5638 error (screenshot below)
❌ DS - No Data Found alerts on both reports
✅ ECV

<img width="1032" alt="Screenshot 2023-06-02 at 2 12 24 PM" src="https://github.com/cityofaustin/atd-data-tech/assets/49930868/97f18f7d-ffd1-41dc-a3e6-fc4ad9fbec57">

**Comment by TracyLinder** [comment]
### Convo w/ ROW during Friday review 6/9/23

- S: EX & ECV look good in test, 
- but DS is still Not Found. She thinks it's due to admin access or part of code could be wrong, not sure. Sree advises to send to PROD, then check it there. No issues expected. 
- Sree requested mtg w/ Jenice to discuss this and #11523 

**ROW: OK with this approach.** 


**EX UAT:** ✅ Looks good

**Comment by TracyLinder** [comment]
### Approved for PROD

**Comment by Nadin-Nader** [comment]
This has been sent to the Amanda Support team to be deployed to PROD under **SCTASK0324682**

**Comment by Sreelatha-BlazeKannan** [comment]
@Nadin-Nader , @TracyLinder - Tested in PROD all looking good; Please verify in PROD so, that this can be closed. Thank you!

**Comment by Nadin-Nader** [comment]
Sent Kyle a message via TEAMS asking for this to be UATd on PROD. Waiting on response

**Comment by Nadin-Nader** [comment]
@Sreelatha-BlazeKannan  @TracyLinder apparently there is an inspector that is missing or got deleted and they need the inspector to be added so Kyle has rejected the UAT on PROD. 
Im waiting on the name of the inspector

**Comment by KyleErich** [comment]
I'm closing this card. The changes reflected in the AC are successfully on PROD, so this issue can be considered resolved. The additional issue identified during the UAT process is being addressed in separate card #12605.
